ksql-engine/src/main/java/io/confluent/ksql/services/SandboxedKafkaTopicClient.java

```diff
@@ -97,7 +99,12 @@ private void createTopic(
             Collections.emptyList()))
         .collect(Collectors.toList());
 
-    createdTopics.put(topic, new TopicDescription(topic, false, partitions));
+    createdTopics.put(topic, new SandboxedTopicDescription(
+        topic,
+        false,
+        partitions,
+        Sets.newHashSet(AclOperation.READ, AclOperation.WRITE)
```

Comment:
why not validate in the sandbox instead of assuming the permissions are valid? we already forward calls like:
```java
        .forward("isTopicExists", methodParams(String.class), sandbox)
```
I think you might be able to leak information otherwise. e.g. if I think a topic 'foo' exists, then I try to run a statement that access foo and it passes validation (otherwise it would fail with "topic does not exist" or something like that) and then fails during real execution with 401

Reply:
How can I reuse the sandbox forwards to describe a topic that returns the fake authorization operations? I looked but I couldn't figure out.